README.md

```diff
@@ -45,7 +45,7 @@ OpenSearch Security is an OpenSearch plugin that offers encryption, authenticati
 
 ## Documentation
 
-Please refer to the [technical documentation](https://opendistro.github.io/for-elasticsearch-docs/docs/security/configuration/) for detailed information on installing and configuring opensearch-security plugin.
+Please refer to the [technical documentation](https://docs-beta.opensearch.org/docs/security/configuration/) for detailed information on installing and configuring opensearch-security plugin.
```

Comment:
Did you confirm with Eli/Kyle that we want to point people at the docs-beta website?

Reply:
we talked about this and yeah, it points to `doc-beta` site

Later we'll have another url without `beta` and we'll update it again